manticore/__main__.py

```diff
@@ -79,51 +78,38 @@ def parse_arguments():
 def main():
     args = parse_arguments()
 
-    m = Manticore(args.argv[0], args.argv[1:], disasm=args.disasm)
-    m.policy = args.policy
-    m.args = args
+    log.set_verbosity(args.v)
+
+
+    #TODO 
+    initial_state = make_initial_state(args.argv[0], argv=args.argv[1:],  disasm=args.disasm)
+    m = Manticore(initial_state, workspace_url=args.workspace,  policy=args.policy)
+
+    #Fixme remove all this...
+    m.coverage_file = args.coverage
 
     if args.data:
         m.concrete_data = args.data
 
-    if args.workspace:
-        m.workspace = args.workspace
-
     if args.profile:
         m.should_profile = args.profile
 
-    if args.dumpafter != 0:
```

Comment:
for all of these that we remove, we should remove them from the --help documentation also

Reply:
We should also add an issue to reimplement using events/plugin